VERSION

```diff
@@ -1 +1 @@
-0.156.0
+0.157.0
```

Comment:
Shouldn't it be `0.156.1`?

Reply:
as I understand its a `feature` and should up minor version

Reply:
Didn't know, Thanks!